spec/dummy/spec/rails_helper.rb

```diff
@@ -29,7 +29,7 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
```

Comment:
I feel like the addition of DatabaseCleaner is unnecessary because there really shouldn't be any tests that actually call the API.